### PR TITLE
Feat/donates list

### PR DESCRIPTION
**Pull request by sabrina34352**
image_2022-07-13_21-12-15
image_2022-07-15_13-25-22
image_2022-07-13_21-12-29

### CODE REVIEW

File: packages/frontapp/src/app/models/helperFunctions/claimHistory.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import { isOverdue } from './converTime';
 import { IMaterial } from '../../components/ListBooks';
-import { StatusTypes } from '../../../../../global-types/src/lib/statusTypes';
-
```
Comment (x8er) [inline packages/frontapp/src/app/models/helperFunctions/claimHistory.ts:4]: i think there is should be an empty row

**Review by x8er — APPROVED** [review]

**Review by x8er — CHANGES_REQUESTED** [review]

File: packages/frontapp/src/app/components/OneDonatedBookPreview/index.tsx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+  white-space: normal;
+
+  @media (max-width: ${dimensions.phone_width}) {
+    position: absolute;
+    width: 100%;
+    margin-top: 40px;
+    height: 150px;
+  }
+`;
+const DonateeName = styled.p`
```
[the remark this answers is not in the document]
Comment (sabrina34352) [reply packages/frontapp/src/app/components/OneDonatedBookPreview/index.tsx:70]: nono, like a person who donates book is called donatee

[the remark this answers is not in the document]
Comment (x8er) [reply packages/frontapp/src/app/components/OneDonatedBookPreview/index.tsx:70]: well, may be you are right, but [here is](https://www.yourdictionary.com/donatee) another meaning. If i incorrect - resolve this conversation

[the remark this answers is not in the document]
Comment (sabrina34352) [reply packages/frontapp/src/app/components/OneDonatedBookPreview/index.tsx:70]: yes i will write Donator

**Review by IvanUglovets — APPROVED** [review]

File: packages/frontapp/src/utils/routes.ts [hunk 3]
```diff
@@ -9,4 +9,7 @@ export enum RoutesTypes {
   DONATES_FROM_USER = '/donates-from-user',
   CREATE_NEW_ITEM = '/create-new-item',
   DONATE_TO_LIBRARY = '/donate-to-library',
+  DONATE_PREVIEW = '/donate',
+  BOOK_PREVIEW = 'item',
```
Comment (x8er) [inline packages/frontapp/src/utils/routes.ts:13]: would that be a problem?

**Review by x8er — APPROVED** [review]

**Pull request merged**